templates/react-native-template-realm-ts/template/app/styles/button.ts

```diff
@@ -0,0 +1,37 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2022 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+import {StyleSheet} from 'react-native';
+import colors from './colors';
+
+export const buttonStyles: StyleSheet.NamedStyles<any> = {
+  button: {
+    paddingVertical: 10,
+    paddingHorizontal: 20,
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginLeft: 20,
```

Comment:
```suggestion
    marginHorizontal: 10,
```

Reply:
This makes the button look nice by itself, otherwise its pushed off screen to the left